src/components/features/SignSection.vue

```diff
@@ -0,0 +1,26 @@
+<script>
+import { defineComponent } from 'vue';
+import ButtonlikeLink from '../common/ButtonlikeLink.vue';
+export default defineComponent({
+    components: { ButtonlikeLink }
+});
+</script>
+
+<template>
+    <div class="sign-section">
+        <ButtonlikeLink class="tertiary">Register</ButtonlikeLink>
+        <ButtonlikeLink class="primary">Log in</ButtonlikeLink>  
```

Comment:
Instead of setting these classes ourselves we should try to create a prop on the component which would set them up for us appropriately. It could be named something like `variant` or whatever you like.
This way we do not need to know the exact class names (and those could change).
It's a good practice to not need to know too much about the details of component's implementation 😄

Reply:
After our yesterday talk about the button case I already knew that I would refactor my buttons and button-like links completely and try to handle choosing variants with props, as you say 👍